ffs/Design.md

```diff
@@ -132,9 +132,8 @@ type ColdStorage interface {
 	// during execution.
 	Store(context.Context, cid.Cid, FilConfig) (FilInfo, []DealError, error)
 
-	// Retrieve retrieves the data using an account address,
-	// and store it in a CAR store.
-	Retrieve(context.Context, cid.Cid, car.Store, string) (cid.Cid, error)
+	// Fetch fetches the cid data in the underlying storage.
+	Fetch(context.Context, cid.Cid, car.Store, string) error
```

Comment:
I like the "unfreeze" naming we use in the storage configs... I wonder if that would be appropriate here and be a cleaner separate meaning from "retrieve"

Reply:
Ha!, yeah the _unfreeze_ name is pretty cool, mostly regarding is understood in a "Cold"Storage scope. I'll give it a thought and might change name in the future, I'll let that suggestion sink a bit more :P